src/operations.rs

```diff
@@ -48,7 +48,7 @@ where
 }
 
 #[cfg(not(windows))]
-fn get_proxy(url: &str) -> Option<Result<ureq::Proxy>> {
+fn get_proxy(url: &str) -> Option<Result<reqwest::Proxy>> {
```

Comment:
Do we even need this, or does `reqwest` handle proxies fully automatically?

Reply:
I removed all the manual proxy handling, my understanding is that reqwest does this transparently in the background.